### ISSUE TITLE
missing SimulatorConfig.profiler_data_path when runing offline.py

### ISSUE THREAD

**Issue by KylinC** [LLMServe/DistServe#7]
missing SimulatorConfig.profiler_data_path when runing offline.py

**Comment by llx-08** [comment]
If your intention is merely to run `offline.py`, you can manually modify the `class OfflineLLM` in the `/distserve/llm.py` file by adding a `SimulatorConfig` instance and setting `is_simulator_mode` to `False`.
like this
```
       # modified
       simulator_config = SimulatorConfig(is_simulator_mode=False,
                                           profiler_data_path='',
                                           gpu_mem_size_gb=70)

        self.engine = LLMEngine(
            model_config,
            disagg_parallel_config,
            cache_config,
            context_sched_config,
            decoding_sched_config,
            simulator_config
        )
```

**Comment by PKUFlyingPig** [comment]
This bug has been fixed in PR(#11), please try again.

**Comment by interestingLSY** [comment]
Closing this issue. Reopen it if you encountered any further issues.
